client/indexer-db/src/models/bucket.rs

```diff
@@ -0,0 +1,81 @@
+use chrono::NaiveDateTime;
+use diesel::prelude::*;
+use diesel_async::RunQueryDsl;
+
+use crate::{schema::bucket, DbConnection};
+
+/// Table that holds the Buckets.
+#[derive(Debug, Queryable, Insertable, Selectable)]
+#[diesel(table_name = bucket)]
+#[diesel(belongs_to(Msp, foreign_key = msp_id))]
+pub struct Bucket {
```

Comment:
Here as well, why not store the bucket's forest root?

Reply:
I imagine that it is not handled yet, but there should be a way of storing files with their metadata, and link those files to this bucket. Also link the BSPs storing that file to the file.